packages/api/endpoints/collections.js

```diff
@@ -26,12 +27,16 @@ const log = new Logger({ sender: '@cumulus/api/collections' });
  * @returns {Promise<Object>} the promise of express response object
  */
 async function list(req, res) {
```

Comment:
Do we also need to add this to the get a single collection? Along with the list ?

Reply:
I didn't think so because we're only using it when we display a list of collections.  I suppose if there was a request to display it on the collection Overview page we could add it.  But idk if it's important now.

Reply:
I say do it later. It's easy.